docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -328,15 +327,15 @@ dictionary that contains all the groups to be managed. Each group's value should
 
 If any of the elements of a group are empty, they may be omitted from the output.
 
-When called with the argument ``--host <hostname>`` (where <hostname> is a host from above), the script must print either an empty JSON hash/dictionary, or a hash/dictionary of variables to make them available to templates and playbooks. For example::
+When called with the argument ``--host <hostname>`` (where <hostname> is a host from above), the script must print either an empty JSON object, or an object containing variables to make them available to templates and playbooks. For example::
```

Comment:
'object' not preceded by 'JSON' is a little ambiguous since that's something else in Python. Maybe moving the 'either' after the JSON object is clearer?
```suggestion
When called with the argument ``--host <hostname>`` (where <hostname> is a host from above), the script must print a JSON object, either empty or containing variables to make them available to templates and playbooks. For example::
```